src/ApiLoader/EnableInTracee.cpp

```diff
@@ -30,7 +30,7 @@ using orbit_user_space_instrumentation::ExecuteInProcess;
 
 namespace {
 
-ErrorMessageOr<absl::flat_hash_map<std::string, ModuleInfo>> GetModulesByPathForPid(int32_t pid) {
+ErrorMessageOr<absl::flat_hash_map<std::string, ModuleInfo>> GetModulesByPathForPid(uint32_t pid) {
```

Comment:
we probably should keep this int32_t, this is linux specific code and uint32_t could end-up being out of bound.

In general uint32_t is fine in the interface but once passed to linux-specific code it should be translated to int32_t so we do not have CHECKs for overflow everywhere.

Reply:
I agree, reverted to int32_t.